integration-tests/esbuild/build-and-test-git-tags.js

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env node
+'use strict'
+
+/* eslint-disable no-console */
+const fs = require('fs')
+const { spawnSync } = require('child_process')
+const assert = require('assert')
+
+const ddPlugin = require('../../esbuild')
+const esbuild = require('esbuild')
+
+const SCRIPT = './git-tags-out.js'
+
+esbuild.build({
+  entryPoints: ['basic-test.js'],
+  bundle: true,
+  outfile: SCRIPT,
+  plugins: [ddPlugin],
+  platform: 'node',
+  target: ['node18'],
+  external: [
+    // dead code paths introduced by knex
+    'pg',
+    'mysql2',
+    'better-sqlite3',
+    'sqlite3',
+    'mysql',
+    'oracledb',
+    'pg-query-stream',
+    'tedious'
+  ]
+}).then(() => {
+  const { status, stdout, stderr } = spawnSync('node', [SCRIPT], {
+    env: { ...process.env, DD_TRACE_DEBUG: 'true' }
```

Comment:
Nit: If you set encoding to be `utf8`, the `stdout` and `stderr` returned here will not be buffers but stings, so you don't need to call `toString()` on them below

```suggestion
    env: { ...process.env, DD_TRACE_DEBUG: 'true' },
    encoding: 'utf8'
```